SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/ReferenceCountedLockingPolicy.h

```diff
@@ -19,35 +21,47 @@ class SPATIALGDK_API UReferenceCountedLockingPolicy : public UAbstractLockingPol
 public:
 	virtual ActorLockToken AcquireLock(AActor* Actor, FString DebugString = "") override;
 
-	// This should only be called during the lifetime of the locked actor
+	// This should only be called during the lifetime of the locked Actor
```

Comment:
nit - punctuation consistency. I think the preference is full stops and the end of sentences. You've done both in different places.